Automattic/jetpack
# WAF: Extract definition of constants from Waf_Runner to Waf_Constants

#### Changes proposed in this Pull Request:

* Moves `define_mode()` and `define_share_data()` from `Waf_Runner` to `Waf_Constants`.
* Renames `Waf_Constants::define_constants()` to `Waf_Constants::define_bootstrap_constants()`, and extracts the three constants that it defines into three distinct methods: `define_waf_directory()`, `define_wpconfig_path()`, and `define_killswitch()`.

#### Other information:


#### Jetpack product discussion



peb6dq-mP-p2

#### Does this pull request change what data or activity we track or use?



No

#### Testing instructions:






* Proof read changes
* Ensure tests pass
* Check for regression in package functionality
